root/static/scripts/edit/URLCleanup.js

```diff
@@ -2674,6 +2674,7 @@ const CLEANUPS: CleanupEntries = {
     },
     clean: function (url) {
       url = url.replace(/^(?:https?:\/\/)?(?:(?:www|mw)\.)?genie\.co\.kr\//, 'https://www.genie.co.kr/');
+      url = url.replace(/^https:\/\/www\.genie\.co\.kr\/detail\/artist(?:Info|Album|Song|Mv)\?xxnm=(\d+)(?:[&#\/].*)?$/, 'https://www.genie.co.kr/detail/artistInfo?xxnm=$1');
       url = url.replace(/^(https:\/\/www\.genie\.co\.kr\/detail\/(?:albumInfo\?ax|artistInfo\?xx|songInfo\?xg|mediaInfo\?xv)nm=\d+)(?:[&#\/].*)?$/, '$1');
```

Comment:
Both of these look at `artistInfo` - should we drop it from the second line? Is it needed twice?

Are there similar multiple pages for album, song, media?

Reply:
I think we can remove it from the second line, yes.
There are tabs on album and song details, but all those other tabs redirect to the artist page, so they're already handled by this.

Reply:
Cleaned up the RegEx, waiting for the tests to pass now.